### DISCUSSION TITLE
Input stack-label styling on Quasar v2

### DISCUSSION THREAD

**Discussion by eladcandroid** [quasarframework/quasar#9031] [General - Components / Directives / etc] [answered]
Here's the stack-label prop on v1:
https://codepen.io/eladcandroid/pen/xxgBZxy

And here on v2:
https://codepen.io/eladcandroid/pen/LYxapwM

How we can get the same styling on v2?

**Comment by eladcandroid** [comment, marked as answer]
OK, Here's my solution:
https://codepen.io/eladcandroid/pen/YzNgqge

Have fun! :)

**Reply by rstoenescu** [reply]
In your first codepen (https://codepen.io/eladcandroid/pen/xxgBZxy) you have Quasar UI v1 with additional CSS, while on the second one (https://codepen.io/eladcandroid/pen/LYxapwM) you have Quasar UI v2 with no additional CSS. Which is why the output differs. Otherwise the CSS hasn't changed in that area at all between Quasar UI v1 and v2.

**Reply by eladcandroid** [reply]
@rstoenescu You're so right.. I didn't notice it and created the CSS by myself... Thanks!!

**Reply by rstoenescu** [reply]
Also, we cannot supply such CSS from within the framework because of the hardcoded background color. But we have this on our TODO list ;)
